### PR TITLE
Sponsorship pages

### PR DESCRIPTION
**Pull request by ascampos**

### CODE REVIEW

File: gatsby-node.js
Comment (dscush) [inline gatsby-node.js:101]: Can you explain what `otherBillNames` is/does?

File: package.json
Comment (dscush) [inline package.json:26]: Extra space:
```suggestion
    "query-string": "^6.13.1",
```

File: package.json
Comment (dscush) [inline package.json:25]: This should be removed - it was replaced with `sass` on line 36 below.

File: src/components/sponsorships/sponsorships.js
Comment (dscush) [inline src/components/sponsorships/sponsorships.js:27]: This will never show the scores in the column on the right because the necessary fields (`recordedVotePercentage` and `score`) are not included in the records in `houseSponsors`. I also noticed that the legislators are sorted alphabetically by first name. There may be a way we can reuse more of the code that puts this data together for the main `all-legislators` page.

File: gatsby-node.js
Comment (ascampos) [reply gatsby-node.js:84]: you're right, I think I was doing it a slightly different way before

File: gatsby-node.js
Comment (ascampos) [reply gatsby-node.js:101]: here I am filtering out bills that are the same just house or senate versions so theyre not displayed twice, but still keeping track of their other name (H---/S---) so we can link to them. It seems like we might have a way to account for this but I wasnt sure

File: src/components/sponsorships/sponsorships.js
Comment (ascampos) [reply src/components/sponsorships/sponsorships.js]: fixed!

File: src/pages/landing/SearchInstructions.js
Comment (dscush) [inline src/pages/landing/SearchInstructions.js:1]: This file was deleted when we reinstated the search form, which happened after you created this PR. So you can delete it from this `upgrades` branch.

**Review by dscush — CHANGES_REQUESTED** [review]
Sorry for the delay on this. I'm going to try to get feedback on PRs/udates within three days going forward, just to keep things moving.

I think the only three things that still need to be addressed are the two comments I left, plus styling the page for an individual bill, and then we'll have the MVP ready to deploy. Let me know if you want me to pick up any of those pieces, and thanks again for taking this on!

**Review by dscush — APPROVED** [review]
To keep things moving, I just went ahead and implemented some small changes. This all LGTM now, pending @JonathanCohn's approval 🚀

**Pull request merged**
